l10n_nl_bank/__manifest__.py

```diff
@@ -1,14 +1,15 @@
+#
 # Copyright 2017 Odoo Experts (<https://www.odooexperts.nl>)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 {
     "name": "Dutch banks list",
     "summary": "Import all Dutch banks with BIC code",
-    "version": "13.0.1.0.0",
-    "category": "Localisation",
+    "version": "13.0.1.1.0",
+    "category": "Localisation/Europe",
     "license": "AGPL-3",
     "author": "Odoo Experts, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/l10n-netherlands",
-    "depends": ["base"],
+    "depends": ["base", "account"],
```

Comment:
What is it that requires account instead of just base in this change?

Reply:
The addition of the account dependency is not specifically related to these changes.
IMO, account should've always been an dependency. Will test further, if not including this dependency returns an error when not installed. Or the module just is useless, because the account fields are unavailable.

Reply:
Test Results:
Module can be installed without errors when "account" is unavailable.
From functional point of view it is pretty useless to install this module without that one.

Reason for adding this was to be inline with similar modules on other localisation languages. Will remove to give users more freedom for use cases.